app/modules/asset_groups/models.py

```diff
@@ -305,6 +305,22 @@ def getter(self):
 
         return getter
 
+    def get_custom_fields(self):
+        return self.__class__.config_field_getter('customFields', default={})(self)
+
+    def get_encounters(self):
+        encounters = self.config and self.config.get('encounters') or []
+        for encounter in encounters:
+            encounter['owner'] = {
+                'full_name': current_user.full_name,
+                'guid': current_user.guid,
+                'profile_fileupload': None,
+            }
+            encounter['createdHouston'] = self.created
+            encounter['updatedHouston'] = self.updated
+        log.warning(f'get_encounters {encounters}')
```

Comment:
warning seems a bit severe, debug seems more appropriate

Reply:
Sorry I just forgot to remove debug messages :joy:

Reply:
Lol, I did wonder :)